samples/Samples.Server/Startup.cs

```diff
@@ -49,8 +48,8 @@ public void ConfigureServices(IServiceCollection services)
             })
             .AddSystemTextJson()
             .AddErrorInfoProvider<CustomErrorInfoProvider>()
-            .AddWebSockets()
             .AddDataLoader()
+            .AddUserContextBuilder(context => new Dictionary<string, object> { { "user", context.User.Identity.IsAuthenticated ? context.User : null } })
```

Comment:
So for this sample we actually need a user context builder where none was defined previously.